src/main/java/com/auth0/client/auth/AuthAPI.java

```diff
@@ -835,4 +836,45 @@ public AuthRequest exchangeCode(String code, String redirectUri) {
         request.addParameter("phone_number", phoneNumber);
         return request;
     }
+
+    /**
+     * Creates a request to exchange the mfa token and one-time password (OTP) to authenticate a user with an MFA OTP Authenticator.
+     *
+     * <pre>
+     * {@code
+     * AuthAPI auth = new AuthAPI("me.auth0.com", "B3c6RYhk1v9SbIJcRIOwu62gIUGsnze", "2679NfkaBn62e6w5E8zNEzjr-yWfkaBne");
+     * try {
+     *      TokenHolder result = auth.exchangeMfaOtp("the-mfa-token”, new char[]{‘a','n','o','t',’p’})
+     *          .execute();
+     * } catch (Auth0Exception e) {
+     *      //Something happened
+     * }
+     * }
+     * </pre>
+     *
+     * @param mfaToken the mfa_token received from the mfa_required error that occurred during login. Must not be null.
+     * @param otp      the OTP Code provided by the user. Must not be null.
+     *
+     * @return a Request to configure and execute.
+     *
+     * @see <a href="https://auth0.com/docs/api/authentication#verify-with-one-time-password-otp-">Verify with one-time password (OTP) API documentation</a>
+     */
+    public AuthRequest exchangeMfaOtp(String mfaToken, char[] otp) {
+        Asserts.assertNotNull(mfaToken, "mfaToken");
```

Comment:
Nit: Remember the second param is used to explain the error in the exception message. We don't need to match exactly the param name, but could make it more readable. e.g.

```suggestion
        Asserts.assertNotNull(mfaToken, "mfa token");
```